nservicebus/rabbitmq/configuration-api.md

```diff
@@ -14,197 +14,242 @@ To use RabbitMQ as the underlying transport:
 
 snippet:rabbitmq-config-useconnectionmanager
 
+WARNING: In Versions 4 and above, the ability to provide a custom connection manager has been removed.
+
 
-### Controlling behavior when broker connection is lost
+## Controlling behavior when the broker connection is lost
 
-By the default the RabbitMQ transport will trigger the on critical error action when it continuously fails to connect to the the broker for 2 minutes. This can now be customized using the following configuration setting: (values must be parsable to `System.TimeSpan`)
+By default, the RabbitMQ transport will trigger the critical error action when it continuously fails to connect to the broker for 2 minutes. The amount of time can be customized using the following configuration settings: (values must be parsable to `System.TimeSpan`)
 
 snippet:rabbitmq-custom-breaker-settings
 
-In Versions 4 and above xml configuration options for controlling lost connection behavior were replaced by code equivalent. 
+In Versions 4 and above, the XML configuration options for controlling lost connection behavior have been replaced by a code equivalent: 
 
 snippet:rabbitmq-custom-breaker-settings-code
 
 
-### Changing routing topology
+## Routing topology
+
 
-By default the RabbitMQ transport create separate exchanges for each message type, including inherited types, being published in the system. This means that polymorphic routing and multiple inheritance for events is supported since each subscriber will bind its input queue to the relevant exchange based on the event types that it has handlers for.
+### Conventional Routing Topology
 
-For less complex scenarios use the `DirectRoutingTopology` that routes all events through a single exchange, `amq.topic` by default. The events will be published using a routing key based on the event type and subscribers will use that key to filter their subscriptions.
+By default, the RabbitMQ transport creates separate [fanout exchanges](https://www.rabbitmq.com/tutorials/amqp-concepts.html) for each message type, including inherited types, being published in the system. This means that polymorphic routing and multiple inheritance for events is supported since each subscriber will bind its input queue to the relevant exchanges based on the event types that it has handlers for.
 
-To enable direct routing use the following configuration:
+NOTE: The RabbitMQ transport doesn't automatically modify or delete existing bindings. Because of this, when modifying the message class hierarchy, the existing bindings for the previous class hierarchy will still exist and should be deleted manually.
+
+
+### Direct Routing Topology
+
+For less complex scenarios, use the `DirectRoutingTopology` that routes all events through a single exchange, `amq.topic` by default. Events will be published using a routing key based on the event type, and subscribers will use that key to filter their subscriptions.
```

Comment:
What's scenarios are `less complex scenarios` ? Can you outline the actual scenarios? What's complex to one person might not be complex to another.